src/Supervisor.ts

```diff
@@ -7,6 +7,9 @@ import connectionPool from "./connectors/ConnectionPool";
 import * as events from "events";
 import DB from "./DB";
 import NodeSSH = require("node-ssh");
+import BaseConnector from "./connectors/BaseConnector";
```

Comment:
Are the BaseConnector and Console imports still necessary or left-over from the previous implementation? If not, let's remove them.